src/grid/widgets/Header.ts

```diff
@@ -29,19 +41,26 @@ export default class Header extends ThemedMixin(WidgetBase)<HeaderProperties> {
 		sorter(id, direction);
 	}
 
+	private _sortRenderer = (column: ColumnConfig, ascending: boolean, sorter: () => void) => {
+		const { columnConfig, sort, filterer, filter = {}, theme, classes } = this.properties;
```

Comment:
Maybe I'm confused but this destructure seems to be taking out a lot more than it uses?

Reply:
Sorted